AndroidNativeKotlinTemplate/app/src/main/java/com/salesforce/androidnativekotlintemplate/QrCodeEnabledLoginActivity.kt

```diff
@@ -0,0 +1,72 @@
+//package com.salesforce.androidnativekotlintemplate
```

Comment:
Is there a reason class and the matching xml are commented out?  Would be a lot easier to automate if they were not.

Reply:
I was wondering here also.  Both of these would only be needed if a user of the template were going to uncomment the QR Code Log In related sources elsewhere.  If that's not done, it seemed nice to omit the need to compile these and have them potentially dangling in the workspace.

Other than running a few extra compilation steps, having them uncommented would be harmless.  Shall we uncomment them?

Reply:
I was mostly thinking about automating but on second thought that isn't going to be possible so I guess it doesn't make much difference.

Reply:
Even if we can't automate the actual login, test_force could make sure everything compiles.
If it is commented out and any API changes, we won't know unless we manually test (by uncommenting the code).
I recommend we put that code in and have the comment to guide them in case they are not planning to use the feature. Or we could move it to its own template.

Reply:
Agreed. I uncommented 99.9% of the code, but kept the configuration with QR Code Login disabled via a couple of key lines just like we do for push notifications.